src/main/java/com/ebivariation/contigalias/controller/ContigAliasController.java

```diff
@@ -45,28 +46,32 @@ public ContigAliasController(AssemblyService assemblyService, ChromosomeService
         this.chromosomeService = chromosomeService;
     }
 
-    @GetMapping(value = "assemblies/{accession}")
+    @ApiOperation(value = "Get an assembly using it's Genbank or Refseq accession.")
+    @GetMapping(value = "assemblies/{accession}", produces = "application/json")
```

Comment:
I just remembered, in other APIs we use endpoints like "/v1/assemblies/{accession}" instead of just "assemblies/{accession}". This comes essential when breaking changes need to be made. Please, add "v1" to all endpoints.